Automattic/wp-calypso
# Sites Management Page: Create a blur background based on site icon for sites thumbnail

#### Proposed Changes

* Currently we are fetching dominant color from the site icon and using it as background color for sites thumbnail. This cause CORS issues when trying to fetch external sites icon. This PR instead solve the issue using a large enough blur of the site icon to get the average color as the backgroud.

#### Testing Instructions

- Go to /sites-dashboard
- Verify background color for sites thumbnail reflects the site icon (both list and tile view)
![image](https://user-images.githubusercontent.com/47489215/183786606-6f1a0524-e668-492d-b781-b63286b27120.png)



*

#### Pre-merge Checklist






Related to #
